client/blocks/conversation-follow-button/index.jsx

```diff
@@ -42,9 +56,14 @@ class ConversationFollowButtonContainer extends Component {
 	};
 
 	render() {
+		const isFollowing = includes(
+			uniq( [ CONVERSATION_FOLLOW_STATUS_FOLLOWING, this.props.defaultConversationFollowStatus ] ),
```

Comment:
I don't think i understand the logic here.  I think we are looking for something like:
```js
const isExplicitlyFollowing = this.props.followStatus === CONVERSATION_FOLLOW_STATUS_FOLLOWING;
const isImplicitlyFollowing =
    this.props.followStatus === CONVERSATION_FOLLOW_STATUS_NOT_FOLLOWING &&
    this.props.defaultConversationFollowStatus === CONVERSATION_FOLLOW_STATUS_FOLLOWING;
const isFollowing = isExplicitlyFollowing || isImplicitlyFollowing;
```

Reply:
is uniq beneficial there?

Reply:
I don't understand the alternative one :)

In the original, I'm making an array of acceptable following statuses, then checking against it. Let me break that out into a separate variable and see if that makes more sense.

I used `uniq` because sometimes `this.props.defaultConversationFollowStatus` is the same as `CONVERSATION_FOLLOW_STATUS_FOLLOWING`, so `includes` has to do slightly less work.